backlogger.py

```diff
@@ -148,12 +132,82 @@ def check_query(data):
         )
 
 
+def render_influxdb(data):
```

Comment:
The max. level of nesting within this function is quite high. It would make sense to extract some parts of it.